src/ontology/components/core.owl

```diff
@@ -78,9 +85,9 @@
     
 
 
-    <!-- http://purl.org/dc/terms/license -->
+    <!-- http://purl.org/dc/elements/1.1/license -->
 
-    <owl:AnnotationProperty rdf:about="http://purl.org/dc/terms/license"/>
+    <owl:AnnotationProperty rdf:about="http://purl.org/dc/elements/1.1/license"/>
```

Comment:
@anitacaron @matentzn Any ideas on why this changed? I didn't change it manually. I thought `dcterms:license` was the correct annotation now.

Reply:
Thanks @anitacaron   
I think we should change this in `core.owl` (i.e., change annotation to `dcterms:license`). I can do the change as part of this PR, but I don't want to interfere with other work you may be doing.

Reply:
Ok